### ISSUE TITLE
Add a check for requires to be sorted

### ISSUE THREAD

**Issue by Heliosmaster** [borkdude/clj-kondo#708]
**Is your feature request related to a problem? Please describe.**
Often namespaced requires grow organically as:

```
(ns foo.bar
(:require 
  [[clojure.spec :as spec]
   [zzz.core :as zzz]
   [clojure.data.csv :as csv]
   [foo.baz]])
```

**Describe the solution you'd like**
Clj-kondo triggers a warning unless namespace requires are sorted
```
(ns foo.bar
(:require
  [[clojure.data.csv :as csv]
   [clojure.spec :as spec]
   [foo.baz]
   [zzz.core :as zzz]])
```
(Note that indentation does not matter for this warning)

If this feature is interesting it might be something that I can take a look into

**Comment by borkdude** [comment]
Sorting has already been mentioned here: https://github.com/borkdude/clj-kondo/issues/229
But it's good to make a separate issue out of it and a separately named linter. I think this linter should be optional (the level is set to `:off`), but it is something I would personally turn on. PR welcome. If you need any help, let me know.
